### ISSUE TITLE
[Suggestion] NoOffhandPlacement

### ISSUE THREAD

**Issue by Akorian** [maruohon/tweakeroo#178]
How about a feature that can disable placing Blocks, torches or the last torch from your offhand. 
I get regularly infuriated when something from my invisible offhand gets placed (especially the last torch).

**Comment by Akorian** [comment]
Any chance this will be added?

**Comment by maruohon** [comment]
Yes, but in general I don't usually add new features directly to the latest (1.16 for example) MC versions, but rather to 1.12.2 first and then port them forward. And currently I'm working on lots of malilib changes in 1.12.2, so no new features are being added for any of the mods before those changes are finished.

**Comment by Akorian** [comment]
Any Chance?
